docs/backend.md

```diff
@@ -80,10 +80,15 @@ properties to the Form object that get written during `#submit`
 At the end of the day, analytics events get dumped into `events.log` and contain
 information like user ID, service provider, user agent, etc.
 
-Event names are strings. Events correspond to methods in the
+Event names correspond to methods in the
 [AnalyticsEvents](../app/services/analytics_events.rb) mixin. We document these
 with YARD so that we can auto-generate
-[documentation on them in our handbook][analytics-handbook]
+[documentation on them in our handbook][analytics-handbook].
+
+> [!NOTE]
```

Comment:
TIL the note syntax!

Reply:
Pretty new feature I think!

https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts